JunitHomework/src/main/java/string/utility/StringToInt.java

```diff
@@ -0,0 +1,29 @@
+package string.utility;
+
+public class StringToInt {
+    private String before;
+
+    public StringToInt() {
+    }
+
+    public String getBefore() {
+        return before;
+    }
+
+    public void setBefore(String before) {
+        this.before = before;
+    }
+
+    public void modifyStringToInt() {
```

Comment:
here for the method modifyStringToInt()  I would choose to have an input parameter.

public String modifyStringNumberToIntNumber(String inputText)